template.adoc

```diff
@@ -0,0 +1,15 @@
+= [short title describing spike]
+John Smith <john.smith@coblox.tech>; Fabien Miller <Fabien@coblox.tech>;
+
+* Author: {authors}
+* Date: {docdate}
```

Comment:
Note it's UTC :)

Reply:
I think {docdate} should be set to something somewhere, otherwise this is going to change when you download the file etc. (The value is passed in by the renderer.)

Talked to @bonomat and we solved it like this:
https://github.com/comit-network/spikes/pull/6/files#diff-5991a65c40c76c0b0446b96e49e23868